Microsoft.WindowsAzure.Storage/tests/UnitTest++

```diff
@@ -0,0 +1 @@
+Subproject commit a3f957c09f99209b54b9b2743bfc97c4fb23703a
```

Comment:
Thanks. That was left over from getting the tests to build. I added UnitTest++ as a submodule. Fixed.